src/screens/SendFunds/06-Validation.js

```diff
@@ -109,6 +110,8 @@ class Validation extends Component<Props, State> {
           if (e && e.statusCode === 0x6985) {
             error = new UserRefusedOnDevice();
           }
+
+          Sentry.captureException(error);
```

Comment:
it should be in the `else` case of the if above. `UserRefusedOnDevice ` is legit